images/cross/run.sh

```diff
@@ -29,6 +29,6 @@ useradd -o -m -g $BUILDER_GID -u $BUILDER_UID $BUILDER_USER 2> /dev/null
 echo "$BUILDER_USER    ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 export HOME=/home/${BUILDER_USER}
 echo "127.0.0.1 $(cat /etc/hostname)" >> /etc/hosts
-[[ -S /var/run/docker.sock ]] && chown $BUILDER_UID:$BUILDER_GID /var/run/docker.sock
+[[ -S /var/run/docker.sock ]] && chmod 666 /var/run/docker.sock
```

Comment:
Do you know why it is not enough for the builder to own the socket?

Reply:
it looks like `osfx` the file system docker uses to bind mount volumes adds extended attributes on a file on a `chown` and that corrupts the socket file. This looks like a bug in docker. the above is a workaround.